src/test/ui/issues/issue-74539.rs

```diff
@@ -0,0 +1,12 @@
+enum E {
+    A(u8, u8),
+}
+
+fn main() {
+    let e = E::A(2, 3);
+    match e {
+        E::A(x @ ..) => {  //~ ERROR `x @` is not allowed in a tuple
+            x //~ ERROR cannot find value `x` in this scope
```

Comment:
Hmm, `x` should be found here?

Reply:
@JohnTitor, this is a consequence of the nature of the fix. But I think you're right, this should be done differently.